Src/BootCamp.Chapter/ILogger.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    interface ILogger
+    {
+        /// <summary>
+        /// Used to log the Action at the current Date and Time
+        /// </summary>
+        /// <param name="action"></param>
+        void ActionLog(string action);
```

Comment:
If you simply swap Log and Action, you get LogAction. However, it should be simplified. It should be LogAction(string message). Action is too ambigous.

Reply:
Agreed.